resource-server/README.md

```diff
@@ -65,5 +66,6 @@ If you see a basic welcome message, then things are working!  Now open a new ter
 [Implicit Flow]: https://developer.okta.com/authentication-guide/implementing-authentication/implicit
 [Okta Angular Sample Apps]: https://github.com/okta/samples-js-angular
```

Comment:
This still isn't public. Should we remove this until it is?

Reply:
This is just the ref, it's not visible in the bulleted list yet